changelog/unreleased/issue-2571

```diff
@@ -0,0 +1,15 @@
+Bugfix: Check contents of unchanged files
+
+We've added an extra check wheter the contents are available in the index when backing
```

Comment:
wether.

Also, I would change "the contents ... unchanged files" to "whether the contents of an unchanged file are actually available in the repository" (not the index).

Reply:
changed.